bitnami/postgresql/README.md

```diff
@@ -552,6 +552,44 @@ $ helm upgrade my-release stable/postgresql \
 
 - The term `master` has been replaced with `primary` and `slave` with `readReplicas` throughout the chart. Role names have changed from `master` and `slave` to `primary` and `read`.
 
+To upgrade to `10.0.0`, it should be done reusing the PVCs used to hold the PostgreSQL data on your previous release. To do so, follow the instructions below (the following example assumes that the release name is `postgresql`):
```

Comment:
I can't select line 541 as it is out of the scope of this review, but this sentence doesn't have sense, can you remove it?